bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -9,13 +9,186 @@
 from cppyy import cppdef, include
 from libROOTPythonizations import gROOT
 from cppyy.gbl import gSystem
+import cppyy
 import cppyy.ll
 
 from ._application import PyROOTApplication
 from ._numbadeclare import _NumbaDeclareDecorator
 
 from ._pythonization import pythonization
 
+byteswap = ""
+
+
+def _CheckEndianessFromTypestr(typestr):
+    import ROOT
+
+    global byteswap
+
+    if len(byteswap) == 0:
+        ROOT.gInterpreter.Declare(
+            """
+std::string PyROOT_getByteswap()
+{
+#ifdef R__BYTESWAP
+    return "<";
+#else
+    return ">";
+#endif
+}
+"""
```

Comment:
This is not nice to me. I would much rather we had a function somewhere in ROOT (e.g. core/foundation) that we can call to get this information without asking the interpreter

Reply:
You are right, it's not optimal. However, I would like to keep this PR restricted to the PyROOT code area.

Therefore, I have put back the `GetEndianess()` function in the C++ `libROOTPythonizations` for now, and then we can think later how to "upstream" this to ROOT.